src/appleseed/renderer/kernel/rendering/progressive/progressiveframerenderer.cpp

```diff
@@ -385,18 +387,19 @@ namespace
         {
             const Spectrum::Mode        m_spectrum_mode;
             const SamplingContext::Mode m_sampling_mode;
-            const size_t                m_thread_count;         // number of rendering threads
-            const uint64                m_max_sample_count;     // maximum total number of samples to compute
-            const double                m_max_fps;              // maximum display frequency in frames/second
-            const bool                  m_perf_stats;           // collect and print performance statistics?
-            const bool                  m_luminance_stats;      // collect and print luminance statistics?
-            const string                m_ref_image_path;       // path to the reference image
+            const size_t                m_thread_count;                     // number of rendering threads
+            const uint64                m_max_average_samples_per_pixel;    // maximum average number of samples to compute per pixel
```

Comment:
I would be fine with using the "spp" abbreviation too :) `m_max_avg_spp` is clear enough IMHO.